app/models/occupation_standard.rb

```diff
@@ -12,6 +12,28 @@ class OccupationStandard < ApplicationRecord
 
   validates :title, presence: true
 
+  def unregistered_clone(creator_id:, organization_id:)
+    begin
+      OccupationStandard.transaction do
+        os = UnregisteredStandard.create!(
+          creator_id: creator_id,
+          organization_id: organization_id,
+          occupation: occupation,
+          title: title,
+          parent_occupation_standard: self,
+        )
+
+        os.occupation_standard_work_processes = occupation_standard_work_processes.map(&:dup)
```

Comment:
Just idle thought bc there's rarely anything to nitpick in your PRs...

I assume this works bc of the specs, but on first glance, I would've thought this might `dup` the `occupation_standard_id`. Presumably the assignment `=` overrides that since you've got specs to cover it. 

I wonder if it's more clear to leverage some rails magic by doing something like `os.work_processes = work_processes`. Doesn't that automatically fill the join table or is that only on `has_and_belongs_to_many` relationships?

Reply:
Yes, it does `dup` the `occupation_standard_id` but then overrides it with the`=`. It doesn't work to do `os.work_processes = work_processes` because we are requiring `hours` now on the join table.  I could theoretically do that with skills, but then we lose the ordering that is also established on the join table.  But if this is bothering you, I could do something like:

```
occupation_standard_work_processes.each do |oswp|
  os.occupation_standard_work_processes.create(
    work_process: oswp.work_process,
    hours: oswp.hours,
    sort_order: oswp.sort_order,
  )
end
```

though the downside of this is that if we add more fields to the join table, then this won't automatically pick them up.

Reply:
Ah! That's right. I forgot about the `hours`. Carry on as-is!